bin/test-dag-tags.py

```diff
@@ -0,0 +1,91 @@
+#!/bin/python3
+from collections import defaultdict
+import subprocess
+
+
+VALID_TAGS = ("impact/tier_1", "impact/tier_2", "impact/tier_3", "repo/bigquery-etl", "repo/telemetry-airflow",)
+REQUIRED_TAG_TYPES = ("impact",)
+
+
+class DagValidationError(Exception):
+    pass
+
+
+def retrieve_existing_airflow_dags():
+    output = subprocess.run(
+        "docker-compose exec web mysql -Ns -h db -u root -psecret airflow -e 'SELECT dag_id, name FROM dag_tag;'",
+        shell=True,
+        capture_output=True
+    ).stdout.decode('ascii').strip()
+
+    dags = defaultdict(list)
+
+    for dag in output.replace("\r", "").split("\n"):
+        dag_name, tag_name = dag.split("\t")
+
+        dags[dag_name] = dags[dag_name] + [tag_name]
+
+    return dags
+
+
+import glob, os
+import re
+
+# for file in glob.glob("*.txt"):
+#     print(file)
+
+def get_dag_tags(dag_dir: str):
+    tag_mapping = {
+        "Tag.ImpactTier.tier_1": "impact/tier_1",
+        "Tag.ImpactTier.tier_2": "impact/tier_2",
+        "Tag.ImpactTier.tier_3": "impact/tier_3",
+        "repo/telemetry-airflow": "impact/tier_3",
+        "repo/bigquery-etl": "repo/bigquery-etl",
+    }
```

Comment:
Maintaining this mapping here seems problematic. I think this should be generated from the enum rather than duplicated here.